cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -2498,4 +2498,36 @@ void destructor_without_block(bool b)
       ClassWithDestructor g;
 }
 
+void destruction_in_switch_1(int c) {
+  switch (c) {
+    case 0: {
+      ClassWithDestructor x;
+      break;
+    }
+  }
+}
+
+void destruction_in_switch_2(int c) {
+  switch (ClassWithDestructor y; c) {
+    case 0: {
+      break;
+    }
+    default: {
+        break;
```

Comment:
Very small thing I just spotted: you might want to align the break here with the one in the other case (similar below)

Reply:
Fixed in 07dd6d5c8dd13bb3a4a869add692d6b78a98cb15